frontend/ui/screensaver.lua

```diff
@@ -150,6 +182,8 @@ local function expandSpecial(message, fallback)
     ret = string.gsub(ret, "%%T", title)
     ret = string.gsub(ret, "%%A", authors)
     ret = string.gsub(ret, "%%S", series)
+    ret = string.gsub(ret, "%%h", time_left_chapter)
+    ret = string.gsub(ret, "%%H", time_left_document)
```

Comment:
Only tangentially related to the PR, but this could be simplified with a replacement table.

e.g.,

```lua
local rep = { ["%c"] = currentpage, ["%t"] = totalpages, ... }
ret = string.gsub(ret, "(%%%a)", rep)
```

Reply:
Thank you! Wasn't aware of this. Made this change in latest iteration.

Reply:
Well, it's kind of my bad for not implementing it this way the first time around, thanks ;).